cloud-formation/membership-app.cf.yaml

```diff
@@ -63,10 +65,8 @@ Resources:
   FrontendAutoScalingGroup:
     Type: AWS::AutoScaling::AutoScalingGroup
     Properties:
```

Comment:
`AvailabilityZones` isn't needed because we supply `VPCZoneIdentifier`. Later on it might make moving things between subnets slightly harder so I'm getting rid of it. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-availabilityzones